### ISSUE TITLE
Add support for additional skos fields

### ISSUE THREAD

**Issue by EnnoMeijers** [netwerk-digitaal-erfgoed/network-of-terms#42]
Add support for skos:exactMatch and skos:related, see Brinkman thesaurus: https://data.bibliotheken.nl/doc/thes/p075598604

**Comment by sdevalk** [comment]
Good one! These indeed aren't available in the API.

The API currently only exposes properties that link to terms in the source itself, e.g. via `skos:broader` and `skos:narrower`. Property `skos:related` would be a natural addition.

A question about `skos:exactMatch`:

The API currently doesn't expose SKOS properties that link to terms in other sources (or other concept schemes), e.g. via `skos:exactMatch`. Do you think this is useful for users when finding/identifying/selecting terms? If so, would it be useful to also include the label (`adoptie`) of the external term (`http://data.beeldengeluid.nl/gtaa/27936`). And: should we also consider supporting similar properties, e.g. `skos:closeMatch` or `skos:narrowMatch`?

**Comment by EnnoMeijers** [comment]
I agree with your point about skos:exactMatch although it could helpful for selecting the right term when seeing definitions for the term from other sources. I think we should not resolve the external uris in order to present their labels, this adds another level of complexity and could cause performance trouble, we should only present the data that is available in the source itself. Same applies indeed for skos:closeMatch and skos:narrowerMatch.

**Comment by ddeboer** [comment]
We support `exactMatch` and `related`. `closeMatch`, `narrowMatch` seem too fine-grained. I’m not sure sources have those available.
